apps/authentication-service/src/shared/auth/authService.test.ts

```diff
@@ -0,0 +1,58 @@
+import fs from "fs";
+import { _testing, getAccessToken } from "./authService";
+import * as authService from "./authService";
+import { exampleToken } from "@/test/testData/users";
+
+const exampleErrorString = "Error: Authentication error";
+
+let tokenBackup: string;
+
+beforeAll(() => {
```

Comment:
I'm not 100% sure if it works this way, but I think you should put these inside your describe so they don't affect more than they should.

Reply:
backing-up and then restoring seems to work. I'm not getting the stale example token in the cache file anymore. I'll move them inside the describe. Thanks!